File: Modul 6/Soal 2/PRAK602-2410817220018-ANGGRAENI DWI ZAHRA.py
ruangan = int(input("Masukkan Jumlah Ruangan: "))
zetsu_awal = list(map(int, input("Masukkan Jumlah Zetsu di Ruangan: ").split()))

zetsu_membelah = [zetsu_awal[i] * (i + 1) for i in range (ruangan)]
print(' '.join(map(str, zetsu_membelah)))

File: Modul 6/Soal 2/PRAK602-2410817220018-ANGGRAENI-DWI-ZAHRA.c
#include <stdio.h>
int main () {
    int ruangan;
    printf("Masukkan Jumlah Ruangan: ");
    scanf("%d", &ruangan);

    int zetsu_awal[ruangan];
    int zetsu_membelah[ruangan];

    printf("Masukkan Jumlah Zetsu di Ruangan: ");
    for (int i = 0; i < ruangan; i++) {
        scanf("%d", &zetsu_awal[i]);

    }    
    for (int i = 0; i < ruangan; i++) {
        zetsu_membelah[i] = zetsu_awal[i] * (i + 1);
    }

    printf("Jumlah Zetsu Setelah Membelah: ");
    for (int i = 0; i < ruangan; i++) {
        printf("%d", zetsu_membelah[i]);
        if (i < ruangan - 1);
        printf(" ");
    }
    return 0;
}